topics/go/packages/encoding/example2/example2.go

```diff
@@ -38,9 +38,9 @@ type (
 func main() {
 
 	// Open the file.
-	file, err := os.Open("./topics/go/packages/encoding/example2/data.json")
+	file, err := os.Open("data.json")
```

Comment:
@ardan-bkennedy do you run this example in your training? And if so do you run it from the root of the training repo or do you run it from this folder? I run it from this folder and always change the path like @flyinprogrammer has done here so I'm fine with the change but I want your input.

Reply:
I never get to this material.